include/service/entities/plugin_service.js

```diff
@@ -1575,6 +1586,13 @@ PluginService.validateDetails = function(details, pluginDirName, cb) {
 	if (!v.validateVersionNum(details.version, true)) {
 		errors.push("An invalid version number ["+details.version+"] was provided.  Must match the form: xx.xx.xx");
 	}
+    
+    //validate pbVersion
+    if(details.pbVersion){        
+        if (!v.validateVersionNum(details.pbVersion, true)) {
+            errors.push("An invalid version number ["+details.pbVersion+"] was provided.  Must match the form: xx.xx.xx");
+        }
+    }
```

Comment:
you can simplify the expression a bit because the version number is not required.
```
if (!v.validateVersionNum(details.pbVersion, false)) {
  errors.push("An invalid version expression ["+details.pbVersion+"] was provided.");
}
```
You do not need the outer ```if (details.pbVersion)``` because we specify in the validation call that it is not required.